src/UKHO.ADDS.Clients.FileShareService.ReadOnly/IFileShareReadOnlyClient.cs

```diff
@@ -17,7 +17,8 @@ public interface IFileShareReadOnlyClient
         Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearchAsync(string searchQuery, CancellationToken cancellationToken);
         Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearchAsync(string searchQuery, int maxAttributeValueCount, CancellationToken cancellationToken);
         Task<IResult<Stream>> DownloadFileAsync(string batchId, string filename);
-        Task<IResult<DownloadFileResponse>> DownloadFileAsync(string batchId, string fileName, Stream destinationStream, long fileSizeInBytes = 0, CancellationToken cancellationToken = default);
+        Task<IResult<Stream>> DownloadFileAsync(string batchId, string fileName, Stream destinationStream, long fileSizeInBytes = 0, CancellationToken cancellationToken = default);
```

Comment:
Reverted this change
```suggestion
        Task<IResult<DownloadFileResponse>> DownloadFileAsync(string batchId, string fileName, Stream destinationStream, long fileSizeInBytes = 0, CancellationToken cancellationToken = default);
```

Reply:
Changes have been made as suggested.